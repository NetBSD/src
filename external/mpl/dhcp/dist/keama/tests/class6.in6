# class declaration config

# options
option dhcp6.mysystem code 1250 = text;
option dhcp6.myversion code 1251 = unsigned integer 16;

# class declaration
class "foobar" {
    match if option dhcp6.mysystem = "version1";
    option dhcp6.myversion 1;
}
