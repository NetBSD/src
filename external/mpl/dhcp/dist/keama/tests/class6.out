{
  # class declaration config
  # options
  "Dhcp6": {
    "option-def": [
      {
        "space": "dhcp6",
        "name": "mysystem",
        "code": 1250,
        "type": "string"
      },
      {
        "space": "dhcp6",
        "name": "myversion",
        "code": 1251,
        "type": "uint16"
      }
    ],
    "client-classes": [
      # class declaration
      {
        "name": "foobar",
        /// from: match if (option dhcp6.mysystem) = 'version1'
        "test": "option[1250].hex == 'version1'",
        "option-data": [
          {
            "space": "dhcp6",
            "name": "myversion",
            "code": 1251,
            "data": "1"
          }
        ]
      }
    ]
  }
}
